src/components/Editor/index.js

```diff
@@ -69,6 +69,16 @@ export default class UnicornEditor extends Component {
   };
 
   render() {
+    const stickerElements = stickers.map((sticker) => (
```

Comment:
Yep, I was actually thinking to have the stickers not hardcoded, but as a prop that you can pass in.